packages/google_sign_in/android/src/main/java/io/flutter/plugins/googlesignin/GoogleSignInPlugin.java

```diff
@@ -230,8 +230,7 @@ public void signIn(Result result) {
      * Gets an OAuth access token with the scopes that were specified during initialization for the
      * user with the specified email address.
      */
-    public void getTokens(Result result, final String email) {
-      checkAndSetPendingOperation(METHOD_GET_TOKENS, result);
```

Comment:
Add a TODO to add back the `checkAndSetPendingOperation()` call once we add proper gating for `getTokens()` in the Dart code.